src/engine/console.cpp

```diff
@@ -506,7 +507,21 @@ bool consolekey(int code, bool isdown)
 
             case SDLK_BACKSPACE:
             {
-                int len = (int)strlen(commandbuf), i = commandpos>=0 ? commandpos : len;
+                size_t commandbuf_len = strlen(commandbuf);
+                if (holding_alt)
+                {
+                    char* last_word = strrchr(commandbuf, ' ');
```

Comment:
Are you sure you're handling the empty string correctly?

Reply:
Not anymore, since you now said smth. Is there a better way to do this?

Reply:
Your code is not very complex, so try it in a separate `.cpp` file in an MWE. There you can call your code on different strings, and see if it crashes.

Reply:
Why should it crash tho, I can't see any reason why it would